packages/components/src/button/README.md

```diff
@@ -1,29 +1,126 @@
 
-Note that this component may sometimes be confused with the Button block, which has semantically different use cases and functionality.
+![Primary button](https://make.wordpress.org/design/files/2019/03/primary-button.png)
 
-## Usage
+#### Text label
+
+All button types use text labels to describe the action that happens when a user taps a button. If there’s no text label, there should be an icon to signify what the button does (e.g. an IconButton component).
+
+![](https://make.wordpress.org/design/files/2019/03/do-link-button.png)
+
+**Do**
+Use color to distinguish link button labels from other text.
+
+![](https://make.wordpress.org/design/files/2019/03/dont-wrap-button-text.png)
+
+**Don’t**
+Don’t wrap button text. For maximum legibility, keep text labels on a single line.
+
+### Hierarchy
+
+![A layout with a single prominent button](https://make.wordpress.org/design/files/2019/03/button.png)
+
+A layout should contain a single prominently-located button. If multiple buttons are required, a single high-emphasis button can be joined by medium- and low-emphasis buttons mapped to less-important actions. When using multiple buttons, make sure the available state of one button doesn’t look like the disabled state of another.
+
+![A diagram showing high emphasis at the top, medium emphasis in the middle, and low emphasis at the bottom](https://make.wordpress.org/design/files/2019/03/button-hierarchy.png)
+
+A button’s level of emphasis helps determine its appearance, typography, and placement.
+
+#### Placement
+
+Use button types to express different emphasis levels for all the actions a user can perform.
+
+![A link, default, and primary button](https://make.wordpress.org/design/files/2019/03/button-layout.png)
+
+This screen layout uses:
+
+1. A primary button for high emphasis.
+2. A default button for medium emphasis.
+3. A link button for low emphasis.
+
+Placement best practices:
+
+- **Do**: When using multiple buttons in a row, show users which action is more important by placing it next to a button with a lower emphasis (e.g. a primary button next to a default button, or a default button next to a link button).
+- **Don’t**: Don’t place two primary buttons next to one another — they compete for focus. Only use one primary button per view.
+- **Don’t**: Don’t place a button below another button if there is space to place them side by side.
+- **Caution**: Avoid using too many buttons on a single page. When designing pages in the app or website, think about the most important actions for users to take. Too many calls to action can cause confusion and make users unsure what to do next — we always want users to feel confident and capable.
+
+## Development guidelines
+
+### Usage
 
 Renders a button with default style.
 
-{% codetabs %}
-{% ESNext %}
 ```jsx
 import { Button } from "@wordpress/components";
 
 const MyButton = () => (
-	<Button isDefault>
-		Click me!
-	</Button>
+
+Click me!
```

Comment:
why was `<Button>` ... `</Button>` removed? Should it be left in for proper react component usage?

Reply:
Whoops. Thanks for noticing that! I've opened #14964 to restore them.

Reply:
Sounds good! Thanks for the speedy response.